src/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+
+ReactDOM.render(<App />, document.getElementById('root'));
+registerServiceWorker();
```

Comment:
Are you still using this? Take your time and remove all the unnecessary lines and files @AGMETEOR

Reply:
Oopps! my bad. I have removed that.